pkg/nsx/services/common/types.go

```diff
@@ -90,6 +90,7 @@ const (
 	AnnotationAttachmentRef            string = "nsx.vmware.com/attachment_ref"
 	AnnotationRestore                  string = "nsx/restore"
 	LabelCPVM                          string = "iaas.vmware.com/is-cpvm-subnetport"
+	AnnotationAssociatedResource       string = "nsx.vmware.com/associatedResource"
```

Comment:
@jianjuns @lxiaopei shouldn't the style of annotation key be consistent with others, i.e. `nsx.vmware.com/associated_resource`?
https://github.com/vmware-tanzu/nsx-operator/blob/6194779a2577a91eac8a3382cf4857d2f345ded5/pkg/nsx/services/common/types.go#L87-L90

Reply:
Good catch, changed to "nsx.vmware.com/associated_resource"